24life/scripts/delayed.js

```diff
@@ -1,7 +1,20 @@
-// eslint-disable-next-line import/no-cycle
-import { sampleRUM } from './lib-franklin.js';
+import { loadScript, sampleRUM } from './lib-franklin.js';
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
-// add more delayed functionality here
+/**
+ * Load the launch library applicable to the domain
+ */
+async function loadAdobeLaunch() {
+  const isProd = window.location.hostname === 'https://www.24hourfitness.com/';
```

Comment:
Does hostname include the protocol and path?

Reply:
Fixed